oneflow/core/framework/infer_util.cpp

```diff
@@ -52,7 +55,10 @@ Maybe<void> TensorDescInferFnUtil::UnchangedDataType(InferContext* ctx) {
     const std::pair<std::string, int32_t>& input_arg = ctx->inputs().at(i);
     if (first_tensor_desc) {
       const TensorDesc& tensor_desc = ctx->InputTensorDesc(input_arg.first, input_arg.second);
-      CHECK_EQ_OR_RETURN(tensor_desc.data_type(), first_tensor_desc->data_type());
+      CHECK_EQ_OR_RETURN(tensor_desc.data_type(), first_tensor_desc->data_type())
+          << Error::TypeError()
+          << "Tensor descriptions should have the same type. Expected "
+          << first_tensor_desc->data_type() << ", but got " << tensor_desc.data_type();
```

Comment:
```suggestion
          << DataType_Name(first_tensor_desc->data_type()) << ", but got " << DataType_Name(tensor_desc.data_type());
```
data_type() 返回的是 enum 类型，输出要转成 string

Reply:
fixed